extraTs/interfaces.ts

```diff
@@ -0,0 +1,13 @@
+interface Permission {
+    [key: string]: {
+        all: string[];
+        read: string[];
+        write: string[];
+        delete: string[];
+    };
+}
+
+interface UserType {
```

Comment:
define it as IUser, no need to add Type at the end.

Reply:
Done changes